foundry.toml

```diff
@@ -25,6 +25,7 @@ src = 'packages/contracts-core/contracts'
 test = 'packages/contracts-core/contracts/test'
 out = 'packages/contracts-core/foundry_artifacts'
 ffi = true
+fs_permissions = [{access = "read-write", path = "./actions"}]
```

Comment:
remove - should be for `contracts-ops`, which already has these permissions i believe